.eas/workflows/pr-request.yaml

```
@@ -1,20 +1,21 @@
+  build:
+    type: build
+    params:
+      platform: ios
+      profile: development
   e2e:
-    name: E2E tests
     type: maestro
-    environment: preview 
-    image: string
+    needs: [build]
+    environment: development 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)